### ISSUE TITLE
v2 breaking changes?

### ISSUE THREAD

**Issue by fantapop** [mileszim/webpurify#15]
Were the only breaking changes in v2 the method names? It appears this way, just want to be sure before I upgrade.  thanks!

**Comment by mileszim** [comment]
The image method name changes and dropped support for nodejs below 4.0. Changed from lowercase to camelCase to match the rest of the functions. I'll edit the release info and README to make it more clear. In this regard, I have been giving it some thought but can't come up with what feels best - sticking with their api conventions (everything lowercase) or making it "javascript" snakeCase. Do you have a preference one way or the other?

**Comment by fantapop** [comment]
I was also feeling the naming convention pain when adding the aim methods!
Their API naming scheme is not ideal but I don't think people will really
care if you match their naming conemtiom. Probably people care most about
it working and Ideally you don't need to make backwards incompatible
changes too often.

Chris

**Comment by mileszim** [comment]
I absolutely agree! Do you think adding a lowercase alias to each method (while not documented, just a dev-friendly addition) would help prevent this confusion?

**Comment by fantapop** [comment]
btw, I made an attempt to upgrade to 2.0.0.  I had to add .default when
using require syntax:

const WebPurify = require('webpurify').default;


Also, it looks like in page anchors for aim methods are no longer working
on README:
https://github.com/mileszim/webpurify#aimimgAccount . It looks like those
still want lowercase.

Also, I was seeing an error trying to use  aimImgCheck

TypeError: webPurify.aimImgCheck is not a function

I'm not sure whats going.  For now I rolled back to my forked version.
